### PR TITLE
[ru] Installation Finished

### PR DESCRIPTION
**Pull request by Leo240**

### CODE REVIEW

**Comment by markstory** [comment]
@Leo240 Can you let me know when this is ready to be merged?

**Comment by Leo240** [comment]
@markstory Yes, sure.

File: ru/installation.rst [hunk 1]
```diff
@@ -67,22 +68,391 @@ CakePHP использует мендежер зависимостей PHP `Comp
...
+   операционные системы скрывают файлы, имена которых начинаются с '.' и
+   таким образом не позволяют копировать их.
+
+#. Убедитесь, что ваша копия CakePHP получена из нашего Git-репозитория, и 
+   корректно распакована, проврив файлы .htaccess.
+   
+   Папка app CakePHP (будет скопирована в корневую папку вашего приложения
+   консолью bake):
+
+   .. code-block:: apacheconf
```
Comment (Leo240) [inline ru/installation.rst]: There is some error in this block

**Comment by Leo240** [comment]
@markstory  It's ready, but as I've described previously there is some problem with apacheconf code-block

File: ru/installation.rst (hunk 1 above)
Comment (markstory) [inline ru/installation.rst]: Looks like the text that follows isn't correctly outdented.

File: ru/installation.rst [hunk 2]
```diff
@@ -67,22 +68,534 @@ CakePHP использует мендежер зависимостей PHP `Comp
...
+
+       <IfModule mod_rewrite.c>
+          RewriteEngine on
+          RewriteRule    ^$    webroot/    [L]
+          RewriteRule    (.*) webroot/$1    [L]
+       </IfModule>
+
+    Папка webroot CakePHP (будет скопирована в корневую папку вашего приложения
+    консолью bake):
```
Comment (markstory) [inline ru/installation.rst]: This text needs to move 1 space to the left I think.

Comment (Leo240) [inline ru/installation.rst]: @markstory You were right. Thank you.

**Comment by Leo240** [comment]
@markstory  I've reindented that piece, what else can be wrong?

**Comment by markstory** [comment]
@Leo240 Doesn't look like there were any issues building the HTML. Travis routinely times out building the PDF versions though.

**Comment by Leo240** [comment]
Ok) Thank you.

**Pull request merged**